src/components/ConnectModal.tsx

```diff
@@ -11,6 +11,29 @@ interface ConnectModalProps {
   closeModal: () => void;
 }
 
+const connectEmail = async () => {
+  const msg = {
+    to: "devashishsood18@gmail.com", // Replace with your recipient
+    from: "devashishsood9@gmail.com", // Replace with your verified sender
```

Comment:
How are we going to deal with this verified sender stuff in prod? If we pay for Twilio does it go away?

Reply:
We can just have it sent from the domain. Since we have reworked the flow and want them to come back to the app anyways. They won't be replying to the mail, just using the link to come look at their app request page

Reply:
Ah I read it wrong, I thought the receiver had to verify.